### PR TITLE
fix japanese texts for translate mini workshop 2

### PR DESCRIPTION
**Pull request by umiremix**
Translated the mini workshop 2
（Continuation from https://github.com/VueVixens/docs/pull/111）

### CODE REVIEW

**Review by ryamakuchi — APPROVED** [review]
@umiremix 
Thank you for the translation! 🎉 

I commented on some points I was anxious in. 👀But overall, there was no problem and LGTM! 😸❤️

**Review by nahokoxxx — APPROVED** [review]
LGTM!

File: jp/workshop/minis/mini2.md [hunk 1]
```diff
@@ -181,29 +181,29 @@ NativeScript Playground で Vue.js の モバイルアプリを作るのはと
...
     .placeholder {
         height: 30%;
     }
 ```
 
-## Call Some Cats
+## 猫を読み込み
 
-Next, add a method called `getACat`. It will look like the `getADog` method. But the Cat API requires an api key, so after we pass the url and the method, we will pass a limit of 1 and a header with the key. After making the request, the `getACat` method will update a variable called `catImage` and log the `catImage.url` that is returned by the request.
+次に、  `getACat`というメソッドを追加します。ベースプロジェクトで作成した`getADog` のようになります。 ただし、Cat APIにはAPIキーが必要なため、urlとメソッドを渡した後、1の制限とキーを含むヘッダーを渡します。 リクエストを行った後、`getACat` メソッドは呼び出された `catImage` という変数を更新し、リクエストによって返された `catImage.url` を記録します。
```
[the remark this answers is not in the document]
Comment (nahokoxxx) [reply jp/workshop/minis/mini2.md]: I agree with @ryamakuchi 👍

**Comment by umiremix** [comment]
@nahokomatsui @ryamakuchi Thank you for your great review!💓

**Comment by jlooper** [comment]
Awesome work!

**Pull request merged**